src/plumbing/options/mod.rs

```diff
@@ -162,6 +164,9 @@ pub enum Subcommands {
         statistics: bool,
         /// The file to create the blame information for.
         file: std::ffi::OsString,
+        /// Only blame lines in the given range.
```

Comment:
I'd mention the '1-based inclusive' part and provide an example.
Git does it, too: 

```
   -L <range>            process only line range <start>,<end> or function :<funcname>
```